### PR TITLE
feature B

### PR DESCRIPTION
**Pull request by nunnunhap**
i implements feature B please review

* add Feature B

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
 </head>
 <body>
     this is master
+    Feature B!!!
```
Comment (nolangjoha) [inline index.html]: change B, lower case

**Review by nolangjoha — CHANGES_REQUESTED** [review]
hahaha!

File: index.html (hunk 1 above)
Comment (nunnunhap) [reply index.html]: Ok.....bye....

**Review by nolangjoha — COMMENTED** [review]
??

**Pull request merged**